third_party/re2/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ if(POLICY CMP0048)
 endif()
 
 project(RE2 CXX)
-include(CTest)
+#include(CTest)
```

Comment:
Just curious, why this change?

Reply:
This is what adds the "CTestDashboardTargets" ("Continuous", "Experimental", "Nightly", "NightlyMemoryCheck") to appear. It is not actually needed to make tests or run CTest. It might be worth suggesting they remove it from re2.